src/mobile/src/main/java/tupperdate/android/App.kt

```diff
@@ -140,16 +141,30 @@ private fun LoggedIn(
             onTitleClick = action.authenticationTesting,
             onRecipeDetailsClick = action.viewRecipe,
         )
+
         is LoggedInDestination.Profile ->
             Profile(
                 userApi = api.users,
                 profile = profile ?: api.users.emptyProfile,
                 onCloseClick = action.back,
                 onSignOutClick = {},
             )
-
+        is LoggedInDestination.ConversationsPage -> ConversationsPage(
+            onRecipeClick = action.home,
+            onProfileClick = action.profile,
+            recipes = listOf(/*TODO fill this parameter*/),
+            conversations = listOf(/*TODO fill this parameter*/)
+        )
         is LoggedInDestination.AuthenticationTesting ->
             AuthenticationTesting(api)
+        is LoggedInDestination.OneConversation -> OneConversation(
+            msgList = listOf(),
+            onReturnClick = action.back,
+            imageOther = "",
+            nameOther = "",
+            otherOnlineStatus = "online",
+            onSendClick = {}
+        )
```

Comment:
Why is there any new destinations ? We should only have new (stateless) `Composable`s